packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -82,6 +96,30 @@ class VideoViewport extends Viewport implements IVideoViewport {
     this.videoElement.remove();
   }
 
+  /**
+   * Sets the video image id to show and hte frame number.
+   * Requirements are to have the imageUrlModule in the metadata
+   * with the rendered endpoint being the raw video in video/mp4 format.
+   */
+  public setVideoImageId(
+    imageIds: string | string[],
+    frameNumber?: number
+  ): Promise<unknown> {
```

Comment:
This feels a bit unusuall, since you say `setVideoImageId` but at the same time it accepts imageIds and also frameNumber. 

How about setVideo?